docs/headless-cms/08-01-add-clusterdocstopic.md

```diff
@@ -0,0 +1,186 @@
+
+```
+NAME                                   PHASE   AGE
+prometheus-concepts                    Ready   59s
+prometheus-guides                      Ready   59s
+```
+
+If a given custom resource is in the `Ready` phase and you want to get details of the created ClusterAssets, such as document names and the location of Minio buckets, run this command:
+
+```
+kubectl get clusterasset -o yaml -l cms.kyma-project.io/docs-topic=prometheus-concepts
+```
+
+The result is as follows:
+
+```
+apiVersion: v1
+items:
+- apiVersion: assetstore.kyma-project.io/v1alpha2
+  kind: ClusterAsset
+  metadata:
+    annotations:
+      cms.kyma-project.io/asset-short-name: docs
+    creationTimestamp: "2019-05-15T13:27:11Z"
+    finalizers:
+    - deleteclusterasset.finalizers.assetstore.kyma-project.io
+    generation: 1
+    labels:
+      cms.kyma-project.io/docs-topic: prometheus-concepts
+      cms.kyma-project.io/type: markdown
+    name: prometheus-concepts-docs-markdown-1b7mu6bmkmse4
+    ownerReferences:
+    - apiVersion: cms.kyma-project.io/v1alpha1
+      blockOwnerDeletion: true
+      controller: true
+      kind: ClusterDocsTopic
+      name: prometheus-concepts
+      uid: 253c311b-7715-11e9-b241-1e5325edb3d6
+    resourceVersion: "6785"
+    selfLink: /apis/assetstore.kyma-project.io/v1alpha2/clusterassets/prometheus-concepts-docs-markdown-1b7mu6bmkmse4
+    uid: 253eee7d-7715-11e9-b241-1e5325edb3d6
+  spec:
+    bucketRef:
+      name: cms-public-1b7mtf1de5ost
+    source:
+      filter: content/docs/concepts
+      metadataWebhookService:
+      - endpoint: /v1/extract
+        filter: \.md$
+        name: assetstore-asset-metadata-service
+        namespace: kyma-system
+      mode: package
+      url: https://github.com/prometheus/docs/archive/master.zip
+  status:
+    assetRef:
+      baseUrl: https://minio.kyma.local/cms-public-1b7mtf1de5ost-1b7mtf1h187r7/prometheus-concepts-docs-markdown-1b7mu6bmkmse4
+      files:
+      - metadata:
+          sort_rank: 1
+          title: Data model
+        name: docs-master/content/docs/concepts/data_model.md
+      - metadata:
+          nav_icon: flask
+          sort_rank: 2
+          title: Concepts
+        name: docs-master/content/docs/concepts/index.md
+      - metadata:
+          sort_rank: 3
+          title: Jobs and instances
+        name: docs-master/content/docs/concepts/jobs_instances.md
+      - metadata:
+          sort_rank: 2
+          title: Metric types
+        name: docs-master/content/docs/concepts/metric_types.md
+    lastHeartbeatTime: "2019-05-15T13:27:24Z"
+    message: Asset content has been uploaded
+    observedGeneration: 1
+    phase: Ready
+    reason: Uploaded
+kind: List
+metadata:
+  resourceVersion: ""
+  selfLink: ""
+```
+
+4. Open the Console UI and navigate to the `docs-ui` view. The new **Prometheus** section with **Concepts** and **Guides** topic groups and alphabetically ordered Markdown documents appear at the bottom of the documentation panel:
```

Comment:
we need to honestly mention here that the content is not perfectly rendered, and the reason is that the source markdown is written for different UI and have custom tags that are not standard for Markdown specification. This is why in case you write documentation for Documentation view you should always follow our guidelines -> and linke there

Reply:
Ok, I added it at the beginning of the doc hoping that this is enough but I can mention it again here as well..